ui_framework/components/button/_button.scss

```diff
@@ -53,6 +63,18 @@
       margin-left: $buttonIconSpacing;
     }
   }
+
+  &.kuiButton--small {
+    .kuiButton__icon {
+      &:first-child:not(:only-child) {
```

Comment:
I'm always torn on this kind of stuff. Generally I try to avoid magical, load order based css that effects naked selectors. I think it's fine though in this case since I doubt anyone would include more than two items within a button.